mocks/secondary_emails_mocks.go

```diff
@@ -0,0 +1,210 @@
+package mocks
```

Comment:
Can we define this more succinctly by embedding other mocks? A lot of passthrough functions here.